services/app-api/handlers/forms/get.test.ts

```diff
@@ -0,0 +1,114 @@
+import { getForm, getLatestForm } from "./get";
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { proxyEvent } from "../../utils/testing/proxyEvent";
+import { StatusCodes } from "../../utils/types/types";
+import { NO_KEY_ERROR_MESSAGE } from "../../utils/constants/constants";
+
+jest.mock("../../utils/dynamo/dynamodb-lib", () => ({
+  __esModule: true,
+  default: {
+    query: jest.fn().mockReturnValue({
+      Items: [
+        {
+          createdAt: 1654198665696,
+          formName: "TEST",
+          formId: "TEST_test-1-22",
+          formJson: {},
```

Comment:
the thing that gets stored here will be the entire `mcpar.json` file, + validation from mcpar/index as we talked about.

Reply:
Makes sense